File: audio.m
pkg load signal % Load the signal processing package

file_path = '/home/sumanth/Desktop/CL/Jazz.wav'; % Ensure to provide the correct file extension
[signal, sample_rate] = audioread(file_path);
time = (0:length(signal)-1) / sample_rate;

figure;
plot(time, signal);
title("Audio signal");
xlabel("Time (s)");
ylabel("Amplitude");
grid on;

File: audio.py
import soundfile as sf
from matplotlib import pyplot as plt
file_path='/home/sumanth/Desktop/CL/week 7'
signal,sample_rate=sf.read(file_path)
time=[i/sample_rate for i in range(len(signal))]
plt.figure(figsize=(10,4))
plt.figure(time,signal)
plt.title("audio signal")
plt.grid(True)
plt.show